source/_components/binary_sensor.rflink.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: "RFLink Binary Sensor"
+description: "Instructions on how to integrate RFLink binary sensors into Home Assistant."
+date: 2018-10-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rflink.png
+ha_category: Binary Sensor
+ha_release: "0.81"
+---
+
+The `rflink` component supports devices that use [RFLink gateway firmware](http://www.nemcon.nl/blog2/), for example, the [Nodo RFLink Gateway](https://www.nodo-shop.nl/nl/21-rflink-gateway). RFLink gateway is an Arduino firmware that allows two-way communication with a multitude of RF wireless devices using cheap hardware (Arduino + transceiver).
+
+First, you have to set up your [RFLink hub](/components/rflink/).
+
+The RFLink component does not know the difference between a `binary_sensor`, a `switch` and a `light`. Therefore all switchable devices are automatically added as `light` by default.
+
+RFLink binary_sensor/switch/light ID's are composed of: protocol, id, switch/channel. For example: `newkaku_0000c6c2_1`.
+
+Once the ID of a binary sensor is known, it can be used to configure it as a binary sensor type in Home Assistant, for example, to hide it or configure a nice name.
+
+Configuring a device as a binary sensor:
+
+{% raw %}
```

Comment:
You should not use these `{% raw %}` tags since it doesn't wrap a Jinja2 template.